### DISCUSSION TITLE
[IOS] [Angular] [PWA] Not working when "public" folder is built with (ng build --prod)

### DISCUSSION THREAD

**Discussion by Kenji94** [ionic-team/capacitor#5369] [Q&A] [answered]
Hi everyone!

I have an angular project with PWA and I added Capacitor to it.
On Xcode when runnning on a simulator device, I saw that on the screen, it was the "public/index.html" that was displayed.
This "public" folder corresponds to the folder "dist" on angular, I think it's generated everytime I run the app on a device?

On angular, to generate the "dist" folder, I run "ng build --configuration=prod". 
The arg "prod" is to minify & uglify the code for security & performance measure, the "public" folder files are minified & uglified.

Running on IOS simulator with the "ng build  --configuration=prod" "public" folder is not working.
But If I replace the "public" folder by the one I get by running "ng build" (without the "prod" arg), the app is running on the simulator!
Could you please help me fellow coders?

**Comment by jcesarmobile** [comment, marked as answer]
public folder is for the native projects only, Capacitor CLI copies whatever is your build folder to public folders in the native projects when you run `npx cap copy` or `npx cap sync`.

For PWA you don't have to do anything, your build folder is what you have to run/load, you don't have to create a public folder.
